### ISSUE TITLE
Collaborative sessions play sound inconsistently

### ISSUE THREAD

**Issue by muganwas** [irllabs/roundaround#263]
I’ve tested a number of scenarios and what I’ve seen is a whole lot of inconsistency. 
- Sometimes when a collaborator joins a session  the owner loses sound
- Sometimes when a collaborator adds a new layer the owner loses sound
- Sometimes the owner doesn’t hear the collaborators steps
- Sometimes the collaborator doesn’t hear their own steps

In most cases all the above occurrences can be fixed by either user adding a new layer and new steps.
In conclusion, there’s possibly an event that is causing loss & restoration of sound
